lib/request.js

```diff
@@ -149,6 +149,17 @@ Object.defineProperties(Request.prototype, {
       return this.raw.url
     }
   },
+  originalUrl: {
+    get () {
+      if (!this.originalUrlValue) {
+        this.originalUrlValue = this.raw.url
+      }
+      return this.originalUrlValue
+    },
+    set (originalUrlValueToSet) {
+      this.originalUrlValue = originalUrlValueToSet
+    }
+  },
```

Comment:
I believe there's no need to have to get/set here, but rather rely on a Symbol that hides the `original` if exists, or the `this.raw.url` otherwise.

Reply:
> hides the `original` if exists, or the `this.raw.url` otherwise.

What do you mean by hide?